src/main/java/seedu/address/logic/parser/RemarkCandidateCommandParser.java

```diff
@@ -26,8 +26,8 @@ public RemarkCandidateCommand parse(String args) throws ParseException {
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
         } catch (IllegalValueException ive) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                    RemarkCandidateCommand.MESSAGE_USAGE), ive);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ive.getMessage())
+                    + "\n" + RemarkCandidateCommand.MESSAGE_USAGE, ive);
```

Comment:
Should probably remove the "\n" 

![image](https://user-images.githubusercontent.com/61644924/140458581-8e2a1ca3-df47-4a1c-bf8a-6c2353e65c0b.png)